main/src/com/google/refine/expr/functions/ToString.java

```diff
@@ -68,7 +69,7 @@ public Object call(Properties bindings, Object[] args) {
                 } 
             }
         }
-        return new EvalError("ToString accepts an object and an optional second argument containing a date format string");
+        return new EvalError(ControlFunctionRegistry.getFunctionName(this) + " accepts an object and an optional second argument containing a date format string");
```

Comment:
Ah, now I see, just the empty function call without args...but can we make these translatable ?

![capture3](https://user-images.githubusercontent.com/986438/36518822-b124c016-174d-11e8-94b6-011dc41e4d45.PNG)

Reply:
Unfortunately not, we would need to work on backend localization and that is a non-trivial task.